### PR TITLE
R code upload

### PR DESCRIPTION
**Pull request by gilbertrebecca**

### CODE REVIEW

File: 520- Final project.R [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+library(usmap)
+library(rigr)
+
+# read in 2018 data for county-level social vulnerability indices from CDC (CDC SVI)
+# based on documentation, the value "-999" represents missing data, so we recoded as NAs
+svi <- read.csv("C:/Users/rebec/OneDrive/UW Graduate School/Programming/SVI_2018_US_county.csv", 
+                na.strings = "-999")
+
+# link to data documentation and dictionary
+# https://www.atsdr.cdc.gov/placeandhealth/svi/documentation/SVI_documentation_2018.html
```
Comment (susancam-eo) [inline 520- Final project.R]: I'll also add info about finding data and documentation into the README

File: 520- Final project.R [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
+# Fit linear regression model for WASHINGTON
+lm_nohsdp_washington <- lm(e_unemp ~ e_nohsdp, data = svi_wa)
+# Print the summary of the linear regression model for Washington
+summary(lm_nohsdp_washington)
+
+
+################# Mapping ###################
+
+# TODO: consider removing from this script
```
Comment (susancam-eo) [inline 520- Final project.R]: I know we had talked about moving all of our functions into a separate script, and this reminder is leftover from that discussion. We can probably just delete the comment

File: 520- Final project.R [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+  
+  qqnorm(data$percent_nohsdp, main = "Q-Q Plot - Percent with no high school diploma", xlab = "Theoretical Quantiles", ylab = "Sample Quantiles")
+  qqline(data$percent_nohsdp)
+  
+  qqnorm(data$percent_disabl, main = "Q-Q Plot - Percent non-institutionalized population with a disability", xlab = "Theoretical Quantiles", ylab = "Sample Quantiles")
+  qqline(data$percent_disabl)
+}
+
+plot_qq_counts(svi)
+plot_qq_percents(svi_state)
```
Comment (susancam-eo) [inline 520- Final project.R]: Oh right, we could do the same for the Washington data like
plot_qq_counts(svi_wa)

File: 520- Final project.R [hunk 4]
```diff
@@ -0,0 +1,364 @@
...
+  par(mfrow = c(2, 3))  # Set up a 2x3 grid for multiple plots
+  hist(data$percent_pov, main = "Histogram - Number below 1.5x poverty line (2018)")
+  hist(data$percent_unemp, main = "Histogram - Number unemployed")
+  hist(data$percent_pci, main = "Histogram - Per capita income")
+  hist(data$percent_nohsdp, main = "Histogram - Number no high school diploma")
+  hist(data$percent_disabl, main = "Histogram - Number non-institutionalized population with a disability")
+}
+
+plot_hist_counts(svi)
+plot_hist_percents(svi_state)
```
Comment (susancam-eo) [inline 520- Final project.R]: Like for the QQ plots, we can also plot histograms for WA state for the sake of completeness

File: 520- Final project.R [hunk 5]
```diff
@@ -0,0 +1,364 @@
...
+
+# explore NAs
+any_na_in_rows <- apply(svi, 1, function(row) any(is.na(row)))
+sum(any_na_in_rows) # just 1 row has any NAs
+
+# discarding data for the single county with significant missing data because we believe the data may not be reliable
+fips_to_discard <- svi[which(any_na_in_rows == TRUE), "fips"] # for 2018, Rio Arriba county in NM discarded
+svi <- filter(svi, fips != fips_to_discard)
+
+# TODO: fix xthis!!
```
Comment (susancam-eo) [inline 520- Final project.R]: I forgot to delete this attempt to fix a problem. We solved it another way, can delete now

**Review by susancam-eo — COMMENTED** [review]
Nice work team, I think we're so so close! I just found a couple more minor things to clean up

**Comment by asratawolde** [comment]
It looks fantastic and makes our work truly stand out. Kudos to the team!

**Pull request merged**